src/test/rpc/Subscribe_test.cpp

```diff
@@ -651,6 +651,110 @@ class Subscribe_test : public beast::unit_test::suite
 
     }
 
+    void testAccountDeliveredAmount()
+    {
+        testcase("Ledger Request, Tx Delivered Amount");
+
+        using namespace test::jtx;
+        using namespace std::chrono_literals;
+
+        Account const alice("alice");
+        Account const bob("bob");
+        Account const carol("carol");
+        auto const gw = Account("gateway");
+        auto const USD = gw["USD"];
+
+        // number of expected 'delivered_amount' available
+        int numExpectedAvailable = 0;
```

Comment:
I hate to come back to this again, but it occurred to me that it would be good to add a `numExpectedNotAvailable` and a corresponding check here and in LedgerRPC_test.cpp

Reply:
Fixed in `Refactor tests and resolve PR feedback 3:`